osu.Game.Rulesets.Taiko/Mods/TaikoModHidden.cs

```diff
@@ -1,23 +1,107 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Game.Beatmaps;
+using osu.Game.Beatmaps.ControlPoints;
+using osu.Game.Configuration;
 using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Objects;
 using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Rulesets.Taiko.Objects.Drawables;
 
 namespace osu.Game.Rulesets.Taiko.Mods
 {
-    public class TaikoModHidden : ModHidden
+    public class TaikoModHidden : ModHidden, IApplicableToDifficulty
     {
         public override string Description => @"Beats fade out before you hit them!";
         public override double ScoreMultiplier => 1.06;
-        public override bool HasImplementation => false;
+
+        [SettingSource("Fade-out Time", "The bigger this multiplier is, the sooner the notes will start fading out")]
+        public BindableNumber<double> FadeOutTimeMultiplier { get; } = new BindableDouble
+        {
+            MinValue = 0.5,
+            MaxValue = 1.5,
+            Default = 1.0,
+            Value = 1.0,
+            Precision = 0.01,
+        };
```

Comment:
Removing this setting for now. If it's added, it should be added in `ModHidden` (the base class) and implemented for all bundled rulesets.